src/zigSetup.ts

```diff
@@ -577,6 +577,37 @@ async function updateStatus(context: vscode.ExtensionContext): Promise<void> {
         });
 }
 
+async function getMirrors(context: vscode.ExtensionContext): Promise<vscode.Uri[]> {
+    let cached = { timestamp: 0, mirrors: "" };
+    const key = "mirror-cache";
+
+    const cachedStr = context.globalState.get<string>(key);
+    if (cachedStr !== undefined) {
+        cached = JSON.parse(cachedStr) as typeof cached;
+    }
```

Comment:
It's possible to store a JSON object in the global state so this could be simplified:
```suggestion
    let cached = context.globalState.get(key, { timestamp: 0, mirrors: "" });
```